packages/vkui/src/hooks/useKeyboard.ts

```diff
@@ -54,3 +58,33 @@ export function useKeyboard(): SoftwareKeyboardState {
 
   return { isOpened };
 }
+
+/**
+ * Проверяет изменение визуального окна.
+ *
+ * Предупреждение: https://caniuse.com/?search=VisualViewport
+ */
+export function useVirtualKeyboard() {
+  const [isOpened, setIsOpened] = React.useState<boolean | undefined>(undefined);
+
+  const { document, window } = useDOM();
+
+  const onResize = () => {
+    if (!window || !document || !window.visualViewport || !document.scrollingElement) {
+      return;
+    }
+
+    const calcIsOpen =
+      window.visualViewport.height * window.visualViewport.scale + 30 <
+      document.scrollingElement.clientHeight;
```

Comment:
мб создадим функцию `forceReflow` в `lib/dom`?

```ts
export const forceReflow = (el: HTMLElement) => {
  void el.clientHeight;
};
```

```suggestion
      forceReflow(document.scrollingElement);
```

Reply:
А зачем

Reply:
чтобы было ясно, что это не просто забытое обращение к `clientHeight`, а именно для вызова `reflow` 

тогда хотя бы коммент оставить надо, чтобы вдруг не удалили

Reply:
А это не просто забытое обращение, там вон знак `<` на строку выше

Reply:
![image](https://github.com/VKCOM/VKUI/assets/5850354/6b9d9ae9-1b6a-4e5a-b616-f98e10017dd9)


..... я не заметил, что это операнд 🤣🤣🤣